### DISCUSSION TITLE
Possible way to change the os baudrate?

### DISCUSSION THREAD

**Discussion by MNS26** [openbouffalo/buildroot_bouffalo#49] [Q&A] [answered]
I have been tinkering with the `uboot_defconfig`, `pine64_ox64_defconfig`/`pine64_ox64_full_defconfig` but just cant get a terminal to propperly show up.

is there a way to have it be at 115200 for example?
i was only able to flash the boot chip because i have a pico, but for some reason the 2nd tty port it has doesnt seem to work.

(used default config for picture)
![image](https://user-images.githubusercontent.com/53867268/224514667-703accac-71f6-45e8-9002-8484702b9130.png)

**Comment by Fishwaldo** [comment]
Looking at M0 output in your screen, D0/linux isn’t booted at all. (The SD card interupts count isn’t incrementing). 

Check your flashing address/groups.

**Reply by MNS26** [reply]
i followed the guide from the readme for all the addresses and group.
![image](https://user-images.githubusercontent.com/53867268/224536786-ab6942ea-1e3b-4df9-b1ca-16da771dc4e2.png)
![image](https://user-images.githubusercontent.com/53867268/224536803-c1020602-09f2-4760-8fb6-321ccfbd51e7.png)

i even tried it with the pre-made ones and that also didnt work.

**Reply by Fishwaldo** [reply]
What USB2UART adapter are you using for D0 output? What pins are you hooked upto?

**Reply by MNS26** [reply]
A rp2040 as a picoprobe. Its the only one i have that does 2M baud
GP14/15 for ttyACM0 and GP16/17 for ttyACM1

**Reply by MNS26** [reply]
i forced uboot to have a baud of 115200 and then it _is_ showing text.
i useda a `Prolific Technology, Inc. PL2303 Serial Port / Mobile Action MA-8910P` i bought from waveshare, sadly that one can only go up to 1.2M baud
![image](https://user-images.githubusercontent.com/53867268/224540310-cc8be6e3-e291-4e42-9027-a961088c24b7.png)

**Reply by MNS26** [reply]
hmmm.... left is running on ttyACM0 at 2M baud, upper right is running on ttyUSB0 at 115200, lower right is running on ttyACM1 at 2M
![image](https://user-images.githubusercontent.com/53867268/224547150-de4bf17d-9457-4c82-afb2-e9bc443af438.png)

im absolutely lost to how its still able to even read ttyACM1 at 2M when the output is at 115.2K

**Reply by MNS26** [reply]
i got it working.
i had to swap the serial port that connected to the serial port 0 and 1 on the picoprobe (swapped the wires)
![image](https://user-images.githubusercontent.com/53867268/224683950-2c62ea80-d186-41b6-9ac5-6d4d85f174ef.png)

EDIT:
after editing the boot config i fixed my issue
but i can only get text to appear when going through the 2M baud window (it doesnt display anything in that window)

**Comment by MNS26** [comment, marked as answer]
got it working (yay).
had to lower the linux console to 115200 in the boot config to get it working (although i cant write at 115200 baud)
![image](https://user-images.githubusercontent.com/53867268/224728603-4a5be308-8990-41e8-bb20-50b132500072.png)

**Reply by MNS26** [reply]
nvm ran into another issue... i cant send character at any other baud rate than 2M 

nvm... i had HW flowcontroll on...
